toolkit/tools/specreader/specreader.go

```diff
@@ -63,13 +66,23 @@ func main() {
 		logger.Log.Panicf("Value in --workers must be greater than zero. Found %d", *workers)
 	}
 
-	err := parseSPECsWrapper(*buildDir, *specsDir, *rpmsDir, *srpmsDir, *distTag, *output, *workerTar, *workers, *runCheck)
+	var toolchainRPMs []string
+	var err error
+	toolchainManifestPath := *toolchainManifest
+	if len(toolchainManifestPath) > 0 {
+		toolchainRPMs, err = schedulerutils.ReadToolchainPackageManifest(toolchainManifestPath)
+		if err != nil {
+			logger.Log.Fatalf("unable to read toolchain manifest file '%s': %s", toolchainManifestPath, err)
+		}
+	}
```

Comment:
Is the check necessary, since we're using `ExistingFile()` on the `toolchain-manifest` argument?
```suggestion
	toolchainRPMs, err := schedulerutils.ReadToolchainPackageManifest(*toolchainManifest)
	if err != nil {
		logger.Log.Fatalf("unable to read toolchain manifest file '%s': %s", toolchainManifestPath, err)
	}
```

Reply:
done

Reply:
I think you missed this one. Or just forgot to commit/push.;)

Reply:
Ok, should be actually included now.